### PR TITLE
Correct compression libs from GIT and JPEG CMake

### PR DESCRIPTION
**Pull request by byrnHDF**

### CODE REVIEW

File: CMakeFilters.cmake [hunk 1]
```diff
@@ -127,8 +127,10 @@ if (HDF4_ENABLE_JPEG_LIB_SUPPORT)
     endif ()
   else ()
     set (HDF4_ENABLE_JPEG_LIB_SUPPORT OFF CACHE BOOL "" FORCE)
-    message (FATAL_ERROR " JPEG support in HDF4 was requested but not found")
+    message (FATAL_ERROR " JPEG support in HDF4 was requested and required but not found")
   endif ()
+else ()
+  message (FATAL_ERROR " JPEG support in HDF4 is required")
```
Comment (bmribler) [inline CMakeFilters.cmake:133]: Do you want to mention --with-jpeg flag?

Comment (byrnHDF) [reply CMakeFilters.cmake:133]: CMake doesn't understand that flag, so no reason to confuse.

Comment (bmribler) [reply CMakeFilters.cmake:133]: Oh, OK, so how does the user fix the issue?

Comment (byrnHDF) [reply CMakeFilters.cmake:133]: What issue? This message instance is because they purposely tried to disable jpeg.

**Review by lrknox — APPROVED** [review]

**Pull request merged**